controllers/limitador_controller.go

```diff
@@ -266,6 +277,12 @@ func (r *LimitadorReconciler) reconcileLimitsConfigMap(ctx context.Context, limi
 		return err
 	}
 
+	// TODO: To be deleted when the upgrade path is no longer needed.
+	err = upgrades.DeleteOldConfigMap(ctx, r.Client(), limitadorObj)
```

Comment:
What about naming this function `UpgradeConfigMapTov070` ?? to make it clear what it does. The implementation is indeed delete the old config map.

Reply:
Thanks. I renamed the function as suggested.